app/jobs/send_client_save_ctc_letter_message_job.rb

```diff
@@ -0,0 +1,33 @@
+class SendClientSaveCtcLetterMessageJob < ApplicationJob
+  def perform(number_of_clients: 0)
+    total_sent = 0
+
+    puts "\n==Sending messages to #{number_of_clients} clients=="
+    Archived::Intake2021.all.limit(number_of_clients).find_each do |intake|
```

Comment:
In theory the `.all` isn't necessary -- the `.limit` is delegated to `.all` by the model, as it happens. Not a blocker.